frontend/src/lib/api.ts

```diff
@@ -6,7 +6,7 @@ import { dayjs } from 'lib/dayjs'
 import { apiStatusLogic } from 'lib/logic/apiStatusLogic'
 import { humanFriendlyDuration, objectClean, toParams } from 'lib/utils'
 import posthog from 'posthog-js'
-import { MessageTemplate } from 'products/messaging/frontend/Library/messageTemplatesLogic'
+import { MessageTemplate } from 'products/messaging/frontend/library/messageTemplatesLogic'
```

Comment:
storybook forced me to make this change

folder on disk is `library`
reference is `Library`

mac os is case insensitive (iirc windows is too, but not all linuxeses)

Reply:
Ah, OK then docker build failed. So I guess this is really `Library` on disk and for some reason Mac sees `library`

did

```
mv products/messaging/frontend/library products/messaging/frontend/temp
git add -A
git commit -m "Temp rename to fix case issue"
mv products/messaging/frontend/temp products/messaging/frontend/Library
git add -A
git commit -m "Fix case sensitivity: library -> Library"
```